cmake/std/atdm/utils/get_known_system_info.sh

```diff
@@ -95,7 +95,7 @@ elif [[ $realHostname == "waterman"* ]] ; then
   hostnameMatch=waterman
   hostnameMatchSystemName=waterman
 
-# Specifically named cts1 systems
+# Specifically named cts1 systems (currently maps to 'serrano' env)
```

Comment:
Just want to make sure: is it intentional to default to the old serrano env unless the user explicitly specifies the custom system config?

Reply:
> Just want to make sure: is it intentional to default to the old serrano env unless the user explicitly specifies the custom system config?

Initially, yes.  Once we get automated builds set up protecting these new 'cts1' builds, then we can change the default system from 'serrano' to 'cts1' and alert the EMPIRE developers (but they can still select the 'serrano' env by putting 'serrano' in the build name).  Then, if no one complains, we can remove the old 'serrano' env all together (so 'serrano' will be ignored if it is in the build name string).

We may not have the time to do such a careful switch over of envs but let's do this time to see how this goes.